### PR TITLE
Minor syntax and formatting changes

### PR DESCRIPTION
**Pull request by celeduc**
* consistent use of quote characters within sample code comments
* Some English syntax errors and typos

### CODE REVIEW

File: docs/contracts.rst [hunk 1]
```diff
@@ -128,7 +128,7 @@ Visibility and Getters
 Since Solidity knows two kinds of function calls (internal
 ones that do not create an actual EVM call (also called
 a "message call") and external
-ones that do), there are four types of visibilities for
```
Comment (chriseth) [inline docs/contracts.rst]: Please also remove the opening parenthesis above.

File: docs/contracts.rst [hunk 2]
```diff
@@ -367,9 +367,9 @@ inheritable properties of contracts and may be overridden by derived contracts.
         }
 
         /// This function is protected by a mutex, which means that
-        /// reentrant calls from within msg.sender.call cannot call f again.
-        /// The `return 7` statement assigns 7 to the return value but still
-        /// executes the statement `locked = false` in the modifier.
+        /// reentrant calls from within "msg.sender.call" cannot call "f" again.
```
Comment (chriseth) [inline docs/contracts.rst]: The backticks are used to format code, I think it is good to keep them unless you have a compelling reason.

File: docs/contracts.rst [hunk 3]
```diff
@@ -1383,6 +1383,6 @@ It is also possible to extend elementary types in that way::
     }
 
 Note that all library calls are actual EVM function calls. This means that
-if you pass memory or value types, a copy will be performed, even of the
+if you pass ``memory`` or value types, a copy will be performed, even of the
```
Comment (chriseth) [inline docs/contracts.rst]: Here, the term "memory" is not used as the name of something but rather the concept and thus I would not code-format it, same goes for storage below.

File: docs/contracts.rst (hunk 2 above)
Comment (celeduc) [reply docs/contracts.rst]: Which is the standard? There are two different styles in this file.

Comment (chriseth) [reply docs/contracts.rst]: Ah I'm sorry, I didn't see that. I would lean slightly towards backticks for code citations (quotes should still be used for anything that is not code). What do you think, @axic?

File: docs/contracts.rst (hunk 1 above)
Comment (celeduc) [reply docs/contracts.rst]: Reverted. Although nested parentheses aren't great, it's the clearest option here.

File: docs/contracts.rst (hunk 3 above)
Comment (celeduc) [reply docs/contracts.rst]: Reverted

File: docs/contracts.rst (hunk 2 above)
Comment (celeduc) [reply docs/contracts.rst]: I applied this throughout this file with the latest commit.

**Review by chriseth — APPROVED** [review]

**Pull request merged**